### DISCUSSION TITLE
Would you want to be able to order PCBs and/or PCB and component kit?

### DISCUSSION THREAD

**Discussion by tjko** [tjko/fanpico#2] [General]
Currently I have no plans offering pre-built fanpico boards (hoping someone would be interested in manufacturing/selling those eventually).

But I'd like feedback if you'd be interested in ordering fanpico PCB and/or a DIY kit that consist the PCB and the components to build a fanpico-0804.

And what would be the preferred platform? Tindie? or something else?

**Comment by dmgeurts** [comment]
Either would be great, have you had any other responses yet?

**Reply by tjko** [reply]
Just a few inquiries so far.   I do have some extra (v1.3) PCBs, so I was thinking to offer a free PCB for first ten people to request one (shipping to US addresses only), as long as they promise to post photo of the finished unit, etc...

**Reply by dmgeurts** [reply]
Nice, but sadly I'm in the UK. 🤣

**Reply by atompkins** [reply]
same here, also UK :(

**Reply by tjko** [reply]
I checked that the PCB weights only about 1.5 oz, so should be able to just stick in an envelope and mail it 'as is'....
I can send couple as a test to UK, email me your address details.

**Reply by dmgeurts** [reply]
That's very kind of you, but I think I may need a slightly altered board. I need to mount it in a 3d printed mount for a water-cooling radiator with 4 fans. I've left the space for the PCB inside, but the fan connectors have to be right-angled ones and not stick outside of the PCB perimeter.

And then there's the thought of a small display... 😄

**Comment by tjko** [comment]
As a test I ordered small batch of boards with SMD parts pre-assembled.  After I have time to test that this works ok, I'll try to setup eBay listing to make these available for purchase:

![fanpico-0804D-pcb-preassembled](https://user-images.githubusercontent.com/1075726/196874615-da98497e-1b04-44b8-86d6-95fe7fe7a702.jpg)
